contracts/neutron_interchain_txs/src/msg.rs

```diff
@@ -16,6 +16,7 @@ pub enum QueryMsg {
     // this query returns acknowledgement result after interchain transaction
     AcknowledgementResult {
         interchain_account_id: String,
+        sequence_id: u64,
```

Comment:
This change will break our integration tests suite here: https://github.com/neutron-org/neutron-integration-tests/blob/main/src/testcases/interchaintx.test.ts#L394

Please open updating PR in integration tests as well.